src/csharp/Grpc.Tools/ProtoCompile.cs

```diff
@@ -318,6 +318,7 @@ protected override string GenerateResponseFileCommands()
                     cmd.AddSwitchMaybe("proto_path", TrimEndSlash(path));
             }
             cmd.AddSwitchMaybe("dependency_out", DependencyOut);
+            cmd.AddSwitchMaybe("error_format", "msvs");
```

Comment:
should we only use msvs  error format when we are on windows?

Reply:
Let's use msvs format even on other platforms so errors on dotnet are consistent. I don't see why dotnet projects on OSX and Linux should behave differently.